less/documentDB.less

```diff
@@ -2577,7 +2577,7 @@ a:link {
   font-style: italic;
 }
 .querydropdown.placeholderVisible::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
-  color: #000;
+  color: #403f3f;
```

Comment:
I think this color is still too close to black:
![image](https://user-images.githubusercontent.com/56978073/228086171-352e7ada-0b3f-41ab-83ab-6458ee504e9f.png)

Reply:
Use #767474 instead:
![image](https://user-images.githubusercontent.com/56978073/228086828-f88ae51f-3b84-4698-b1a6-fcb772ed54b1.png)

It will meet the 4.5:1 contract ratio requirement.

Reply:
It also should have at least 3:1 contrast ratio in comparison with background. I will check both and make a commit for this.